packages/react-core/scripts/createSubpaths.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Re-export subpath modules into the package root directory for ease of access.
+ */
+const { readdirSync, readFileSync, writeFileSync, mkdirSync } = require('fs');
+const { resolve, join } = require('path');
+
+const isIndex = (fileName) => fileName.match(/^index\./);
+
+const updatePaths = (filePath, subPathName) => {
+  const contents = readFileSync(filePath, { encoding: 'utf8' });
+  if (filePath.includes('.map')) {
+    return contents.replaceAll('../../../', `../`);
+  } else {
+    return contents.replaceAll('./', `../dist/esm/${subPathName}/`);
+  }
+};
+
+['next', 'deprecated', 'components', 'layouts', 'helpers'].forEach((subPathName) => {
```

Comment:
Can you move this script into the root directory scripts folder, and then use a config file or pass in the parameters for the subpaths.  This way we have it at one place instead of 2 copies.  (e.g.  `yarn run createSubpaths.js --subpaths deperecated,components,layouts,helpers`   or  create a **subpath.json** file that has the subpaths as json objects.  
`
{
   "subpaths": ["next", "deprecated", "components", "layouts", "helplers]
`

then in the script file just read them in by requiring the json file.

Reply:
Yeah I planned on cleaning it up a bit either as part of this PR or a followup, just wanted to get something up quick so that other people could test it and actually make sure that it works before I put more time into this approach.